hccpy/hcc.py

```diff
@@ -134,7 +136,10 @@ def profile(self, dx_lst, age=70, sex="M",
         dx_set = {dx.strip().upper().replace(".","") for dx in dx_lst}
         cc_dct = {dx:self.dx2cc[dx] for dx in dx_set if dx in self.dx2cc}
         cc_dct = V22I0ED2.apply_agesex_edits(cc_dct, age, sex)
-        hcc_lst = self._apply_hierarchy(cc_dct, age, sex)
+        if no_hierarchy:
```

Comment:
Thanks for creating this option. But I have a bit different thought. I will leave my comments.